contrib/aws/aws-sdk-go-v2/aws/aws.go

```diff
@@ -101,6 +123,324 @@ func (mw *traceMiddleware) startTraceMiddleware(stack *middleware.Stack) error {
 	}), middleware.After)
 }
 
+func extractResourceNameFromParams(requestInput middleware.InitializeInput, awsService string) (resourceNameKey string, resourceNameValue string) {
+	switch awsService {
```

Comment:
Not a go expert but should we initialize `resourceNameKey, resourceNameValue` first or provide a default case here?

Reply:
Yeah, we should have a constant defined for each of the cases we support and a default case that reports some sort of error if it's not supported.

Reply:
Good point! I added a new commit :)